src/SecureElementReader.App/Views/Dialogs/VerificationInfoDialog.axaml.cs

```diff
@@ -45,26 +49,24 @@ protected override void OnOpened()
                             TextWrapping = TextWrapping.Wrap
                         });
                     }
-
                 }
                 sp.Children.Add(new TextBlock
                 {
-                    Text = $"{Properties.Resources.PossibleSolution}:",
+                    Text = $"{possibleSolution}:",
                     FontWeight = FontWeight.Bold,
                     TextWrapping = TextWrapping.Wrap
                 });
                 sp.Children.Add(new TextBlock
                 {
-                    Text = $"{Properties.Resources.InstallRCAandICAurl}:",
+                    Text = $"{installRCAandICAurl}:",
```

Comment:
This `installRCAandICAurl` variable should be named either:

1. installRCAandICAUrl or
2. installRcaAndIcaUrl